src/System.Runtime.InteropServices/pkg/System.Runtime.InteropServices.pkgproj

```diff
@@ -11,12 +11,18 @@
     <ProjectReference Include="..\ref\4.0.20\System.Runtime.InteropServices.depproj">
       <SupportedFramework>net46;netcore50</SupportedFramework>
     </ProjectReference>
+    <ProjectReference Include="..\ref\4.1\System.Runtime.InteropServices.depproj">
+      <SupportedFramework>net462</SupportedFramework>
+    </ProjectReference>
     <ProjectReference Include="..\ref\System.Runtime.InteropServices.csproj">
-      <SupportedFramework>net462;netcoreapp1.0</SupportedFramework>
+      <SupportedFramework>net463;netcoreapp1.0</SupportedFramework>
     </ProjectReference>
-    <ProjectReference Include="..\src\System.Runtime.InteropServices.csproj">
+    <ProjectReference Include="..\src\redist\net462\System.Runtime.InteropServices.depproj">
```

Comment:
Can you file an issue to consolidate this with the other depproj?  Feel free to assign to me.